build/controller/Dockerfile

```diff
@@ -12,7 +12,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 ARG ARCH
-FROM golang:1.17.11
```

Comment:
Can you bump it to 1.19? as we should make it consistent with:

https://github.com/kubernetes-sigs/scheduler-plugins/blob/b86321ab439e564cd0ff53f7b90393c587c75475/go.mod#L3